OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -1306,11 +1306,11 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 
 							if (ibfNode.Any())
 								rh.Value.Nodes.RemoveAll(n => ibfNode.Contains(n));
-								Console.WriteLine("The 'ImagesByFullness' property from the removed RenderHarvester trait has been");
```

Comment:
Be sure to leave a message when you do. I saw some of your pull request being silently rebased so they weren't merged already, because no one noticed.